### ISSUE TITLE
Mistake in Probabilistic Matrix Factorization Documentation?

### ISSUE THREAD

**Issue by jonathanng** [pymc-devs/pymc#2377]
Looking at https://pymc-devs.github.io/pymc3/notebooks/pmf-pymc.html, it reads:

> As expected, our MCMC sampler provides lower error on the training set. However, it seems it does so at the cost of overfitting the data. This results in a decrease in test RMSE as compared to the MAP, even though it is still much better than our best baseline.

However, if you look at the actual numbers:

> PMF MAP training RMSE: 3.98510
> PMF MAP testing RMSE:  4.23455

vs

> Posterior predictive train RMSE: 3.92308
> Posterior predictive test RMSE:  4.18124

It seems like MCMC is actually better, right?

**Comment by twiecki** [comment]
good catch, probably some underlying changes and rerunning produced different results.

**Comment by junpenglao** [comment]
@twiecki for reproducibility we should set the seed for the notebooks.

**Comment by twiecki** [comment]
@junpenglao agreed.

**Comment by jonathanng** [comment]
I just noticed this notebook/documentation isn't on the 3.1 version of the documentation:

https://pymc-devs.github.io/pymc3/examples.html#applied

**Comment by twiecki** [comment]
The rendered file in the docs can be found here: http://docs.pymc.io/notebooks/probabilistic_matrix_factorization.html

The underlying NB can be found here:
https://github.com/pymc-devs/pymc3/blob/master/docs/source/notebooks/probabilistic_matrix_factorization.ipynb

**Comment by deega3000** [comment]
The above links giving Page not found error. 
Where to find doc in following link https://docs.pymc.io/notebooks/probabilistic_matrix_factorization.html

**Comment by eigenfoo** [comment]
@deega3000 the PMF example notebook was removed in an earlier release, it's no longer available on the docs.

**Comment by deega3000** [comment]
@eigenfoo 

One of my live project is based on that notebook. The documentation was also very helpful. Unfortunately I don't have it offline. Is there a way to see it again.

**Comment by junpenglao** [comment]
You can copy it from the snapshot at this PR https://github.com/pymc-devs/pymc3/pull/2999
